src/main/webapp/WEB-INF/jsp/editDisplayPreferences.jsp

```diff
@@ -51,20 +51,20 @@
                                         <td>
                                             <c:choose>
                                                 <c:when test="${ts.topic.subscriptionMethod == 1}">
-                                                    <input type="checkbox" disabled="disabled" checked="checked" value="true" name="subscribed_${status.index}"/>
+                                                    <input role="checkbox" aria-labelledby="SubscriptionName" type="checkbox" disabled="disabled" checked="checked" value="true" name="subscribed_${status.index}" aria-checked="true"/>
```

Comment:
I think it may not be necessary, as upon inspecting it I did not find any function it provided.

Also, I made a mistake here, since my aria-labelledby tag only points to the first element of that id.